gallery/src/util/jsUtil.ts

```diff
@@ -12,3 +12,32 @@
 export function ensureArray<T>(items: T | T[]): T[] {
   return Array.isArray(items) ? items : [items];
 }
+
+// Quick and dirty removal of any comment appearing at the beginning of the content, as it is assumed to be
+// a license which is not helpful to the example display. Both HTML-syntax and C-syntax comments are removed
+export const removeLicense = (content: string) =>
+  content
+      .replace(/^<!--(.|\n)*?-->\s*\n?/, '')
+      .replace(/^\/\*(.|\n)*?\*\/\s*\n?/, '');
+
+// Copies a string to the clipboard.
+export const copyTextToClipboard = (textToCopy: string) => {
+  // navigator clipboard api needs a secure context (https)
+  if (navigator.clipboard && window.isSecureContext) {
+    // navigator clipboard api method'
+    navigator.clipboard.writeText(textToCopy);
+  }
+  else {
+    // text area method
+    const textArea = document.createElement('textarea');
+    textArea.value = textToCopy;
+    // make the textarea not visible
+    textArea.style.position = 'fixed';
+    textArea.style.opacity = '0';
+    document.body.appendChild(textArea);
+    textArea.focus();
+    textArea.select();
+    document.execCommand('copy');
+    document.body.removeChild(textArea);
```

Comment:
Might want to set focus back to whatever it was on before this method executed.  As it is now it'll probably set it back to the `body` since you're removing the focused element.

Reply:
> Might want to set focus back to whatever it was on before this method executed. As it is now it'll probably set it back to the `body` since you're removing the focused element.

I actually removed the focus() as it was not required. 
82608d9d729df00c7f41d74640faba7edf16da20